go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/Cray-HPE/hms-xname
+
+go 1.17
+
+require github.com/Cray-HPE/hms-base/v2 v2.0.0-00010101000000-000000000000
+
+require (
+	github.com/fsnotify/fsnotify v1.4.9 // indirect
+	github.com/hashicorp/go-cleanhttp v0.5.1 // indirect
+	github.com/hashicorp/go-retryablehttp v0.5.4 // indirect
+	golang.org/x/sys v0.0.0-20191005200804-aed5e4c7ecf9 // indirect
+)
+
+replace github.com/Cray-HPE/hms-base/v2 => ../hms-base
```

Comment:
This line needs to get removed after https://github.com/Cray-HPE/hms-base/pull/16 is merged and tagged. Then need to run:
```bash
go mod tidy
go mod vendor
```